### PR TITLE
Add README for setting up reduced privilege AWS #11

### PR DESCRIPTION
**Pull request by jesselumarie**
Adds instructions for a lower privilege AWS user based on [this comment](https://github.com/StreisandEffect/streisand/issues/461#issuecomment-272302800) by @MartinLeedotOrg.

https://github.com/StreisandEffect/discussions/issues/11

### CODE REVIEW

**Comment by MartinLeedotOrg** [comment]
Thanks for making this useful :)

**Comment by jesselumarie** [comment]
@cpu / @nopdotcom is there anything else I should do to get this PR ready to review?

File: documentation/AWS.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+IAM Policy Setup
+-----
+
+* Log onto AWS and navigate to the IAM policy panel
+![](screenshots/AWS/IAMPolicy1.png?raw=true)
+
+* Create a IAM Policy
+![](screenshots/AWS/IAMPolicy2.png?raw=true)
+
+* Write a name and description in for the policy, paste in the below policy document, then create the policy by clicking the button in the lower right corner.
```
Comment (alimakki) [inline documentation/AWS.md]: Write a name and description in for the policy -> Write a name and description for the policy 

(the 'in' can be removed)

**Comment by alimakki** [comment]
I'll give this a quick test later on in the evening, but otherwise it's looking quite good. Thanks @jesselumarie for the PR.

**Comment by jesselumarie** [comment]
@nopdotcom I'll give it a go tonight!

**Comment by jesselumarie** [comment]
@nopdotcom I resized those PNGs — let me know if anything else needs to be updated!

**Pull request merged**